docs/sphinx/reference-libobs-graphics-math.rst

```diff
@@ -36,4 +36,4 @@ Helper functions/macros for graphics math.
 .. function:: float rand_float(int positive_only)
 
    Generates a random floating point value (from -1.0f..1.0f, or
-   0.0f..1.0f if *positive_only* is set).
+   0.0f..1.0f if *positive_only* is set).
```

Comment:
Did this whitespace get changed? Was that intentional?

Reply:
Unintentional. Fixed.